src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -46,6 +46,18 @@
 register_cli_argument('resource feature list', 'resource_provider_namespace', CliArgumentType(options_list=('--namespace',), required=False, help=_PROVIDER_HELP_TEXT))
 register_cli_argument('resource feature', 'feature_name', CliArgumentType(options_list=('--name', '-n'), help='the feature name'))
 
+register_cli_argument('resource policy', 'resource_group_name', arg_type=resource_group_name_type, completer=get_resource_group_completion_list)
+register_cli_argument('resource policy', 'policy_definition_name', options_list=('--name', '-n'), help='name of policy definition')
+register_cli_argument('resource policy create', 'rules', help='JSON formatted string or a path to a file with such content')
+register_cli_argument('resource policy create', 'display_name', help='display name of policy definition')
+register_cli_argument('resource policy create', 'description', help='description of policy definition')
+register_cli_argument('resource policy assignment', 'policy_assignment_name', options_list=('--name', '-n'), help='name of the assignment')
+register_cli_argument('resource policy assignment', 'resource_id', help='id of the associated resource')
```

Comment:
>resource_id [](start = 53, length = 11)

Should this use the 'id_part' parameters? The custom logic suggests that resource-group + name | id are required. If this is the case, it should follow the pattern we've used elsewhere for enabling the '--ids' parameter.